packages/react/src/components/Table/TablePropTypes.js

```diff
@@ -225,6 +225,8 @@ export const I18NPropTypes = PropTypes.shape({
   toolbarTooltipLabel: PropTypes.string,
   /** tooltip text for filter row icon button */
   filterRowIconDescription: PropTypes.string,
+  /** overflow menu text callback for truncated filter tags */
+  filterTagsOverflowMenuText: PropTypes.func,
```

Comment:
We should open this up to both string and function. Graphite does not use function translations.

Reply:
@davidicus Thanks for the heads up! I added string for translation as well. Also, I fixed text overflow issue.